sentences/pl/_common.yaml

```diff
@@ -10,23 +10,23 @@ responses:
 lists:
   color:
     values:
-      - in: "biel | biał(y|o)"
+      - in: "biel | biał(y|o|e)"
         out: "white"
-      - in: "czerń | czarn(y|o)"
+      - in: "czerń | czarn(y|o|e)"
         out: "black"
-      - in: "czerwień | czerwon(y|o)"
+      - in: "czerwień | czerwon(y|o|e)"
         out: "red"
-      - in: "pomarańcz | pomarańczow(y|o)"
+      - in: "pomarańcz | pomarańczow(y|o|e)"
         out: "orange"
-      - in: "żółt(y|o)"
+      - in: "żółt(y|o|e)"
         out: "yellow"
-      - in: "zieleń | zielon(y|o)"
+      - in: "zieleń | zielon(y|o|e)"
         out: "green"
-      - in: "niebiesk(i|o)"
+      - in: "niebiesk(i|o) | niebieskie"
```

Comment:
Can't we just use a `ie` in the original form like `niebiesk(i|o|ie)`?

Reply:
Yes, later I noticed that I can do that. I will try to improve it today